dashboards/google-app-engine/gae-app.json

```diff
@@ -0,0 +1,1162 @@
+        "height": 4,
+        "widget": {
+          "title": "System",
+          "sectionHeader": {
+            "subtitle": "",
+            "dividerBelow": false
+          }
+        }
+      },
+      {
+        "yPos": 87,
+        "width": 48,
+        "height": 4,
+        "widget": {
+          "title": "Datastore",
+          "sectionHeader": {
+            "subtitle": "",
+            "dividerBelow": false
+          }
+        }
+      },
+      {
+        "yPos": 127,
+        "width": 48,
+        "height": 4,
+        "widget": {
+          "title": "Memcache",
+          "sectionHeader": {
+            "subtitle": "",
+            "dividerBelow": false
+          }
+        }
+      },
+      {
+        "width": 24,
+        "height": 15,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "filter": "metric.type=\"appengine.googleapis.com/system/instance_count\" resource.type=\"gae_app\"",
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "perSeriesAligner": "ALIGN_MEAN",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [
+                        "resource.label.\"module_id\"",
+                        "resource.label.\"project_id\""
+                      ]
+                    }
+                  },
+                  "unitOverride": "",
+                  "outputFullDuration": true
+                },
+                "tableTemplate": "",
+                "minAlignmentPeriod": "60s"
+              }
+            ],
+            "metricVisualization": "NUMBER",
+            "columnSettings": [
+              {
+                "column": "module_id",
+                "visible": true
+              },
+              {
+                "column": "project_id",
+                "visible": true
+              },
+              {
+                "column": "value",
+                "visible": false
+              }
+            ],
+            "displayColumnType": false
+          },
+          "title": "App Versions"
+        }
+      },
+      {
+        "xPos": 24,
+        "width": 24,
+        "height": 15,
+        "widget": {
+          "title": "Incidents",
+          "incidentList": {
+            "monitoredResources": [
+              {
+                "type": "gae_app"
+              }
+            ],
+            "policyNames": []
+          }
+        }
+      }
+    ]
+  },
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "project_id",
+      "stringValue": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "module_id",
+      "stringValue": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "version_id",
```

Comment:
Can this be removed? This is not used in the json spec.

Reply:
This is a pinned dashboard filter, not a template variable. It allows users to quickly apply this filter universally to the dashboard.